seclang/directives.go

```diff
@@ -29,29 +29,30 @@ import (
 
 // DirectiveOptions contains the parsed options for a directive
 type DirectiveOptions struct {
-	Waf    *coraza.Waf
-	Config types.Config
-	Opts   string
-	Path   []string
+	Waf       *coraza.Waf
+	Config    types.Config
+	Arguments string
```

Comment:
You could consider `Args`, it's somewhat idiomatic e.g. `os.Args`

Reply:
You are right, I will fallback to Args. I was also thinking about preprocessing them into a slice. 
SecDirective 123 456: []string{"123", "456"}
SecDirective 123 "456 789": []string{"123", "456 789"}